azure-iot-device/samples/advanced-hub-scenarios/register_x509.py

```diff
@@ -23,17 +23,11 @@
 
 async def main():
     async def register_device():
-        x509_cert = X509(
-            cert_file=os.getenv("X509_CERT_FILE"), key_file=os.getenv("X509_KEY_FILE")
-        )
-        x509_security_client = X509SecurityClient(
+        x509_cert = X509(cert_file=os.getenv("X509_CERT_FILE"), key_file=os.getenv("X509_KEY_FILE"))
```

Comment:
>X509 [](start = 20, length = 4)

assuming that this can accept a passphrase too

Reply:
yes this can accept passphrase as well. 

---
In reply to: [294543899](https://github.com/Azure/azure-iot-sdk-python-preview/pull/109#discussion_r294543899) [](ancestors = 294543899)